### DISCUSSION TITLE
.

### DISCUSSION THREAD

**Discussion by garthmahns** [Security-Onion-Solutions/securityonion#2914] [Unsupported Versions]

**Comment by weslambert** [comment]
If you could, please provide a little more detail about your issue.  Unfortunately, we don't have much information to work with at the moment. 😄

**Reply by garthmahns** [reply]
lol, sorry Wes. I was trying to delete this because I had found a partial solution in a different thread. Still having an issue though with a master/sensor deployment. the SO install log says the sensor installed with no errors or issues, the salt manager pulled it in, but the manager on the site is not seeing the sensor and is not getting alerts. 

The sensor is outside of my network, all firewall rules are open to allow traffic between the devices. Nat is set properly as well.
